lib/pulsar/client/transport/curl.py

```diff
@@ -1,11 +1,5 @@
 import logging
 
-try:
-    from galaxy import eggs
-    eggs.require("six")
```

Comment:
This probably ought to remain until galaxyproject/pulsar#85 is finished, and then the Pulsar client lib should be updated from Pulsar once it's done.